ucfcs/GrowthPlus
# Julio/parent portal realm

This PR implements data from realm into the UI components for the parent portal, child portal and landing page. 
As of now, we are just retrieving data from realm and it is not as robust as I would like, catch and try statements are still needed and a re-implementation of the lessons and roadmap schemas are needed as well to stay consistent with scoring system. 

We are eliminating the report schema in favor of additional fields in the child schema, it became redundant to have the report schema. Instead, we can add more fields to child schema and centralize the data, the less queries the better. 

To test: 
Build and run the app and play around navigating between screens, make sure nothing breaks and the back buttons are
working as they should. 
Make sure the components have the sample data from realm, there are some components that just placeholders with no effect, 
the child cards with data are clickable and so are the add cards. 

Note:
Create child and remove child are not implemented yet.
